utils/python/CIME/XML/component.py

```diff
@@ -16,15 +16,12 @@ def __init__(self, infile):
         initialize an object
         """
         files = Files()
-
-        # use xmllint to validate infile
-        xmllint = find_executable("xmllint")
-        if xmllint is not None:
-            # all components refer to the same schema so referencing config_drv_file is okay
-            schema = files.get_schema("CONFIG_DRV_FILE")
-            if schema is not None:
-                logger.debug("Checking file %s against %s"%(infile, schema))
-                run_cmd_no_fail("%s --noout --schema %s %s"%(xmllint, schema, infile))
+        schema = files.get_schema("CONFIG_DRV_FILE")
+        if schema is not None:
+            # not checking schema on external components yet
+            cimeroot = get_cime_root()
+            if  cimeroot in os.path.abspath(infile):
```

Comment:
This only applies to xml files within cime for now, when the component models catch up we can apply to them as well.